### ISSUE TITLE
CO1 - Listing pages filtered by term (ie. semester) are not highlighted in menu as current page

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#499]
Listing pages filtered by term ID are not highlighted in menu. They seem to be missing the active-trail class, but they do receive the screen-reader-only "(current)" text.

**Sample of default Course Outlines page (Works as expected)**
- *URL:* course-outlines
![image](https://cloud.githubusercontent.com/assets/1927902/16854989/55fe72d0-49e1-11e6-9e73-7989cee3df8d.png)

**Sample of default Course Outlines page filtered by term 54 (ie. Winter 2015)**
- *Page:* course-outlines/!term
![image](https://cloud.githubusercontent.com/assets/1927902/16855023/7399e298-49e1-11e6-82d4-e9d3d893bb7f.png)

**Sample of custom Course Outlines page created with separate URL (Works as expected)**
- *Page:* course-outlines/sofam/
- *View pane:* filtered by term 29 (ie. SOFAM) via Pane settings
![image](https://cloud.githubusercontent.com/assets/1927902/16855037/91eedab4-49e1-11e6-9c8d-ca519e00858c.png)

**Sample of custom SOFAM Course Outlines page filtered by term 60 (ie. Winter 2016)**
- *Page:* course-outlines/sofam/!term
- *View pane:* filtered by term 29 (ie. SOFAM) via Pane settings
- *Term:* Set to 60 (i.e Winter 2016)

![image](https://cloud.githubusercontent.com/assets/1927902/16855068/bb664bd4-49e1-11e6-885a-11d5fca12045.png)

**Comment by brharp** [comment]
fixed in 7.7.0

**Comment by mmafe** [comment]
This issue appears to be occurring on profiles as well as course outlines. Is this fix meant to apply to all menu items where the sidebar is filtered by a taxonomy term?

**Comment by mmafe** [comment]
Never mind -- figured this out.

They had added the menu as a "Block" but it needed to be added as a "Menu Block" for the styling to work out.

**Comment by brharp** [comment]
To prevent this we could either:


a) apply the proper styling to Drupal's core menu blocks, or


b) disable Drupal's core menu blocks (an option provided by the menu block module)


________________________________
